bigframes/dataframe.py

```diff
@@ -610,7 +629,14 @@ def __repr__(self) -> str:
         column_count = len(pandas_df.columns)
 
         with display_options.pandas_repr(opts):
-            repr_string = repr(pandas_df)
+            import pandas.io.formats
+
+            options = (
```

Comment:
Can we rename to `to_string_kwargs`? I got confused and thought we were modifying a global dictionary.

Reply:
renamed